Gemfile

```diff
@@ -1,16 +1,26 @@
 source 'https://rubygems.org'
 
-gemspec
-
 group :development do
-  gem 'vagrant',      git: 'https://github.com/mitchellh/vagrant.git'
-  gem 'rake'
+  gem 'vagrant',
+      :git => 'git://github.com/mitchellh/vagrant.git',
+      :ref => 'v1.8.1'
+
+  # test dependencies
+  gem 'minitest'
+  gem 'mocha'
+
   gem 'cucumber', '~> 2.1'
   gem 'aruba', '~> 0.13'
   gem 'komenda', '~> 0.1.6'
   gem 'launchy'
   gem 'gem-compare'
   gem 'mechanize'
+
+  # build tool
+  gem 'rake', '10.4.2'
```

Comment:
There seems to be a bug in the latest version of Rake in combination with Minitest where test file loading will print a lot of warnings. Setting a fixed version for Rake will fix that.